charms/interface_kubernetes_cni.py

```diff
@@ -0,0 +1,74 @@
+import hashlib
+import os
```

Comment:
Silly suggestions for this whole file is to type some of these interfaces

```suggestion
import ipaddress
import os

from ops import CharmBase, Relation, Unit
from pathlib import Path
from typing import Optional
```